src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java

```diff
@@ -57,8 +58,9 @@ else if (config.database.maintainInterval < 0)
             Platform platformInstance;
             switch (platform.type) {
                 case "mturk":
-                    //TODO somone needs to implement mturk ... NOOOONE
-                    throw new IllegalArgumentException("Nonono we cannot do this now.");
+                    //FIXME remove the sandbox url - but I am to paranoid
+                    platformInstance = new MturkPlatform(platform.user, platform.password, "https://mechanicalturk.sandbox.amazonaws.com/", platform.name);
```

Comment:
if someone just enters by accident the productive url we have a major problem. I dont want that. so I keep the hardcoded url.